odxtools/parameters/parameterwithdop.py

```diff
@@ -49,7 +49,8 @@ def _resolve_snrefs(self, diag_layer: "DiagLayer") -> None:
         super()._resolve_snrefs(diag_layer)
 
         if self.dop_snref:
-            self._dop = diag_layer.diag_data_dictionary_spec.data_object_props[self.dop_snref]
+            spec = diag_layer.diag_data_dictionary_spec
+            self._dop = spec.data_object_props.get(self.dop_snref) or spec.structures.get(self.dop_snref)
```

Comment:
Both data_object_props and structures can be used here, Both inherit from DopBase

Reply:
possibly. let's fix them on an "as needed" basis. I also consider it to be possible that something like this can happen with the short name resolution of different kinds of objects...

BTW: the short name resolution is not entirely correct at the moment: `NamedItemList` transforms short names into valid python identifiers, so short names that start with a digit get an underscore prefix and will thus currently not be properly resolved. (we would need something like
```python
obj = [ x if x.short_name == snref for x in objlist ][0]
```
but I don't consider this to be a big enough issue to take immediate action...